prototype/index.html

```diff
@@ -337,6 +338,10 @@
 
             var e = document.getElementById('heritage-search');
 
+            function goBack() {
+              window.history.back();
```

Comment:
`window.history.back()` might not work on all browsers (really old ones or opera mini, but I think it'll be fine for the ones we're testing). http://caniuse.com/#feat=history